src/app/forms/components/weights/weights.component.html

```diff
@@ -0,0 +1,107 @@
+<ng-container *ngIf="isEditable">
```

Comment:
```suggestion
<ng-container *ngIf="isEditing; else viewOnly">
```
and change line 66 to
```
<ng-template #viewOnly>
```